lib/main.js

```diff
@@ -189,6 +200,21 @@ export default {
     });
   },
 
+  async getProgram(Linter, configurationPath) {
+    let program;
+    const tsconfigPath = path.resolve(path.dirname(configurationPath), 'tsconfig.json');
```

Comment:
Oops, my bad. My brain totally read `tsconfig.json` as `tslint.json` haha. Disregard my suggestion =]. I will create a task to eventually mimick the search up logic for the `tsconfig.json` file at a later date.